docs/reference/commandline/service_create.md

```diff
@@ -761,6 +762,16 @@ $ docker service create --name dns-cache -p 53:53/tcp -p 53:53/udp dns-cache
 $ docker service create --name dns-cache -p 53:53/udp dns-cache
 ```
 
+### Provide credential specs for managed service accounts (Windows only)
+
+This option is only supported on a daemon running on Windows. The `--credential-spec` must be in the format
```

Comment:
Let's remove the reference to the daemon. As I understand it, this applies to the containers being run, not the daemon that the client is communicating with to set up the service.